### DISCUSSION TITLE
How to make music?

### DISCUSSION THREAD

**Discussion by LNLenost** [tfdsoft/famidash#161] [Q&A] [answered]
Hey! I wanted to collaborate to the project by making some music. I have not much experience, but may someone help me? THX!

**Comment by zephyrside** [comment, marked as answer]
you need:
- a computer running Windows or Linux
- WINE (Linux only)
- Dn-FamiTracker 0.5.0.1
- a copy of the repo (i recommend visual studio code for editing)
